testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/ejb/security/callerprincipal/GetCallerPrincipalTestCase.java

```diff
@@ -215,7 +215,8 @@ public void testStatelessLifecycle() throws Exception {
             Assert.assertEquals(OK + "stop", results.getSlsb("predestroy"));
             return null;
         };
-        Util.switchIdentitySCF("user1", "password1", callable);
+        //Util.switchIdentitySCF("user1", "password1", callable);
+        callable.call();
```

Comment:
Unless I'm missing something the callable should just be removed and its code inlined.